tutorial/cloudshell_tutorial.md

```diff
@@ -52,18 +52,16 @@ While everything provisioned in this guide should fall within GCP's free tier, i
 
 ### Setting up GCP
 
-In addition to a GCP account, you'll need to create a **GCP Project** to follow
-this guide. You can [create one](https://console.cloud.google.com/projectcreate)
-in the GCP console. You'll need the *Project ID* later. You can see a list of
-your projects in the [cloud resource
-manager](https://console.cloud.google.com/cloud-resource-manager). It may take a
-few minutes for the project to be created.
+Create or select the project you'd like to use throughout this tutorial below.
+
+<walkthrough-project-billing-setup></walkthrough-project-billing-setup>
```

Comment:
This will appear as below, prompting the user to create a project + associate billing with it.

![image](https://user-images.githubusercontent.com/2080520/63113546-9b43b600-bf47-11e9-86da-0500d29a9b9b.png)